### ISSUE TITLE
Typo in Exchange.php breaking currencies

### ISSUE THREAD

**Issue by langabi** [ccxt/ccxt#898]
I believe that a typo in Exchange.php is preventing it from correctly loading currency symbols. Specifically, vendor/ccxt/ccxt/php/Exchange.php line 945 is ``'code' => $market['base'],`` instead of ``'code' => $market['quote'],``, as one can see from eg vendor/ccxt/ccxt/build/ccxt.browser.js line 1922. Making that change fixes the issue, but I'm not confident enough in the code base to submit a pull request yet - sorry!

- OS: Linux
- Language: PHP
- CCXT version: 1.10.451
- Exchange: any, but Bitstamp as example
- Method: load_markets()

```
$bs = new \ccxt\bitstamp();
$bs->load_markets();
var_dump($bs->currencies);
```

```
array (size=6)
  'LTC' => 
    array (size=2)
      'id' => string 'BTC' (length=3)
      'code' => string 'LTC' (length=3)
  'ETH' => 
    array (size=2)
      'id' => string 'EUR' (length=3)
      'code' => string 'ETH' (length=3)
  'XRP' => 
    array (size=2)
      'id' => string 'USD' (length=3)
      'code' => string 'XRP' (length=3)
  'BCH' => 
    array (size=2)
      'id' => string 'BTC' (length=3)
      'code' => string 'BCH' (length=3)
  'BTC' => 
    array (size=2)
      'id' => string 'USD' (length=3)
      'code' => string 'BTC' (length=3)
  'EUR' => 
    array (size=2)
      'id' => string 'USD' (length=3)
      'code' => string 'EUR' (length=3)
```

Note the missing entry for 'USD'

**Comment by kroitor** [comment]
> Making that change fixes the issue, but I'm not confident enough in the code base to submit a pull request yet

I'd say you seem to be confident enough! ) Thanks for your attention! This is obviously an example of a bad copypaste bug. Indeed, editing that single line in Exchange.php will fix the issue, so if you can, please submit a PR, I would merge it wight away! Thx again for reporting this issue!

**Comment by kroitor** [comment]
I'll fix it right now, no worries.

**Comment by kroitor** [comment]
Fixed in 1.10.416. Thx again!
